deployments/helm/nvidia-device-plugin/templates/_helpers.tpl

```diff
@@ -269,3 +269,19 @@ We convert this to JSON so that it can be included and converted to an object us
 {{- $_ := set $options "addMigMonitorDevices" ( ne ( (include "nvidia-device-plugin.allPossibleMigStrategiesAreNone" . ) | trim ) "true" )  -}}
 {{- mustToJson $options -}}
 {{- end -}}
+
+{{- define "nvidia-device-plugin.filepathJoin" -}}
```

Comment:
Thoughts @klueska @elezar ?

Reply:
We can do the same thing by using:
```
            path: {{ clean ( join "/" ( list .Values.nvidiaDriverRoot "/etc/nvidia-imex" ) ) | quote }}
```
at the call site. I would rather use built-ins if possible.